### PR TITLE
CE-256 First iteration over direct upload section

### PR DESCRIPTION
**Pull request by MuchoLucho**
# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [ ] Make sure that all commits follow the conventional commit [specification](https://github.com/mendersoftware/mendertesting/blob/master/commitlint/grammar.md) for the Mender project.

The majority of our contributions are fixes, which means your commit should have the form below:

```
fix: <SHORT DESCRIPTION OF FIX>

<OPTIONAL LONGER DESCRIPTION>

Changelog: <USER-FRIENDLY-CHANGE-DESCRIPTION> or <None>
Ticket: <TICKET NUMBER> or <None>
```

- [x] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

**Review by TheMeaningfulEngineer — CHANGES_REQUESTED** [review]
Misses a new section in the server installation (the evaluation with docker compose) on how to enable variables for the server to work this way.


UPDATE: Make no sense to complicate docker compose with this + it wont be visible
It should just link to the variable in the helm chart once it's updated.

File: 07.Server-installation/09.Direct-upload/docs.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    category: docs
+---
+
+!!! Support for Direct Upload is available exclusively in the Mender Enterprise plan in the On Premise offering.
+!!! See [the Mender features page](https://mender.io/product/features?target=_blank)
+!!! for an overview of all Mender plans and features.
+
+The Direct Upload feature provides a Mender channel to upload artifacts directly to the Mender artifact storage, processing it and putting it in the correct place and structure, without going through any other Mender component like the API gateway or deployments service.
+This is an API/CLI feature only, and you can see the progress in the Mender Web UI only after the process completes successfully.
+This feature drastically reduces the time to upload the Mender artifact to the Mender artifact storage, especially when uploading large artifacts (1TB for example), while also reducing the load on the Mender server infrastructure.
```
Comment (tranchitella) [inline 07.Server-installation/09.Direct-upload/docs.md:13]:
```suggestion
The Direct Upload feature provides a Mender channel to upload artifacts directly to the Mender artifact storage, processing it and putting it in the correct place and structure, without going through any other Mender component like the API gateway or deployments service.
This is an API/CLI feature only, and you can see the progress in the Mender Web UI only after the process completes successfully.
This feature drastically reduces the time to upload the Mender artifact to the Mender artifact storage, especially when uploading large artifacts (1TB for example), while also reducing the load on the Mender server infrastructure.
```

File: 07.Server-installation/09.Direct-upload/docs.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Prerequisites
+
+<!--AUTOVERSION: "version %"/ignore-->
+!!! This feature is available starting from the Mender Product bundle version 3.6.3
+
+* The `mender-cli` tool. You can find information about how to download it and the installation steps in the [Downloads section](../../10.Downloads/docs.md#mender-cli) from this documentation.
+
+* To use the direct upload feature, the user needs to have at least the Deployments Manager and Releases Manager roles.
```
Comment (tranchitella) [inline 07.Server-installation/09.Direct-upload/docs.md:22]:
```suggestion
* To use the direct upload feature, the user must have at least the Deployments Manager and Releases Manager roles.
```

File: 07.Server-installation/09.Direct-upload/docs.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+* Enable the feature on the server by applying the [helm charts](https://github.com/mendersoftware/mender-helm#complete-list-of-parameters) with:
+    * `deployments.directUpload.enabled` to `true`
+    * `deployments.directUpload.skipVerify` to `true`
+
+
+## Upload process
+
+For uploading an artifact you will need to have a session token, this could be solved by running the following command:
```
Comment (tranchitella) [inline 07.Server-installation/09.Direct-upload/docs.md:31]:
```suggestion
For uploading a Mender Artifact you will need to have a session token. You can get one by running the following command:
```

File: 07.Server-installation/09.Direct-upload/docs.md (hunk 1 above)
Comment (TheMeaningfulEngineer) [reply 07.Server-installation/09.Direct-upload/docs.md:13]: Couldn't commit this one, github prevented saying no change exists

**Review by TheMeaningfulEngineer — APPROVED** [review]

**Pull request merged**